package.json

```diff
@@ -661,17 +665,16 @@
         "shelljs": "^0.8.5",
         "source-map-support": "^0.5.21",
         "ts-node": "^10.9.2",
-        "typescript": "^3.8.3",
+        "typescript": "^5.0.3",
         "vsce": "^1.87.0",
-        "vscode-azureextensiondev": "^0.5.0",
-        "webpack": "^4.46.0",
-        "webpack-cli": "^3.3.12"
+        "webpack": "^5.74.0",
+        "webpack-cli": "^4.6.0"
     },
     "dependencies": {
+        "@microsoft/vscode-azext-utils": "^2.0.0",
```

Comment:
This replaces vscode-azureextensionui.

Getting the remaining dependencies to be compatible with these new ones and each other was not easy.  Next I'll update @microsoft/vscode-azext-utils to its latest.